crates/config/src/prover.rs

```diff
@@ -19,16 +23,16 @@ pub enum ProverType {
 }
 
 impl ProverRunnerConfig {
-    pub fn fill_values_with_prompt(
-        self,
-        prev_atlantic_api: &str,
-    ) -> anyhow::Result<ProverRunnerConfig> {
+    pub fn fill_values_with_prompt() -> anyhow::Result<ProverRunnerConfig> {
+        let prev_atlantic_api = env::var("MADARA_ORCHESTRATOR_ATLANTIC_API_KEY")
+            .unwrap_or_else(|_| "ATLANTIC_API_KEY".to_string());
```

Comment:
`README` file should mention the need of `ATLANTIC_API_KEY` (if it doesn't already)